baselines/teams.md

```diff
@@ -381,19 +421,21 @@ available in GCC or DoD tenants.
 
 ### Implementation
 
-To ensure that teams email integration is disabled:
+All the settings in this section are configured in the [**Microsoft Teams admin
+    center**](https://admin.teams.microsoft.com).
+
+#### MS.TEAMS.7.1v1 instructions:
 
 1.  Sign in to the **[Microsoft Teams admin
     center](https://admin.teams.microsoft.com).**
 
-2.  Select **Teams** -\> **Teams Settings**.
+2.  Select **Teams** > **Teams Settings**.
 
 3.  Under the **Email integration** section, set **Allow users to send
```

Comment:
The actual configuration setting on the admin center screen is worded slightly different from the instructions.

![image](https://github.com/cisagov/ScubaGear/assets/107076927/32624266-6a2c-43f5-ad30-6da2239dd058)